### PR TITLE
Task 4 implement enter name & join game room page

### LINKED ISSUE #4: Task: Implement Enter name & Join game room page
image

Implement enter name & join game room page
- [ ] Title
- [ ] Please enter name field
- [ ] name field input
- [ ] Enter code field
- [ ] code field input

### PR DESCRIPTION
**Pull request by ewhanhan**
Notes:

- couldnt use Figma 1 to 1, as Figma specs are a bit off and dont work with the native sizing of React Native UI. Ex. Figma UI containers are 566x1018, which is different than iPhone UI size.
- input/button dont do anything at this point

### CODE REVIEW

**Comment by jsong336** [comment]
@EwhanHan should rebase or merge master to the branch

**Comment by EwhanHan** [comment]
> @EwhanHan should rebase or merge master to the branch

rebased to master

File: app/Pages/JoinGamePage.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { SafeAreaView, TextInput, View, Text, StyleSheet } from 'react-native';
+import GlobalStyles, { globalStyling } from '../styles/global';
+import EnterButton from '../component/EnterButton';
+
+const JoinGamePage = () => {
```
Comment (jsong336) [inline app/Pages/JoinGamePage.tsx:6]: We should have all screens into app/screen and name it JoinGameScreen.tsx

**Review by jsong336 — CHANGES_REQUESTED** [review]
Could you move Pages/JoinGamePage.tsx to screen/JoinGameScreen.tsx and rename the component. Seems good otherwise 👍

**Comment by jhwan7** [comment]
@jsong336 update the branch rule, so it always matches master

File: app/Pages/JoinGamePage.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  return (
+    <SafeAreaView style={[GlobalStyles.safeAreaView, Styles.joinGameContainer]}>
+      <Text style={[Styles.text, Styles.headerText]}>Guess Whom</Text>
+      <Text style={[Styles.text, Styles.subText]}>Please Enter Your Real Name</Text>
+      <TextInput style={Styles.inputBox}></TextInput>
+      <Text style={[Styles.text, Styles.subText]}> Please Enter Room</Text>
+      <TextInput style={Styles.inputBox}></TextInput>
+      <EnterButton />
```
Comment (jhwan7) [inline app/Pages/JoinGamePage.tsx:14]: start putting these in a string file that just contains the text

**Review by jhwan7 — CHANGES_REQUESTED** [review]

File: app/Pages/JoinGamePage.tsx (hunk 2 above)
Comment (jhwan7) [reply app/Pages/JoinGamePage.tsx:14]: "please Enter room" etc...

Comment (EwhanHan) [reply app/Pages/JoinGamePage.tsx]: changes made

**Pull request closed without merging**